feature-assets/src/main/java/io/novafoundation/nova/feature_assets/presentation/views/priceCharts/formatters/RealPriceChangeTextInjector.kt

```diff
@@ -0,0 +1,43 @@
+package io.novafoundation.nova.feature_assets.presentation.views.priceCharts.formatters
+
+import android.widget.TextView
+import io.novafoundation.nova.common.resources.ResourceManager
+import io.novafoundation.nova.common.utils.formatting.formatAsPercentage
+import io.novafoundation.nova.common.utils.setDrawableStart
+import io.novafoundation.nova.common.utils.setTextColorRes
+import io.novafoundation.nova.feature_assets.R
+import io.novafoundation.nova.feature_currency_api.domain.model.Currency
+import io.novafoundation.nova.feature_currency_api.presentation.formatters.formatAsCurrency
+
+interface PriceChangeTextInjector {
+    fun format(fromValue: Float, toValue: Float, textView: TextView)
+}
+
+class RealPriceChangeTextInjector(
+    private val resourceManager: ResourceManager,
+    private val currency: Currency
+) : PriceChangeTextInjector {
+
+    override fun format(fromValue: Float, toValue: Float, textView: TextView) {
+        val change = toValue - fromValue
+        val changeInPercent = if (fromValue != 0f) {
+            (change / fromValue) * 100f
```

Comment:
I suggest we use `Fraction` whenever possible. Like `(change / fromValue).fractions`